docs/manual/about/under-the-hood/guarantees.md

```diff
@@ -20,15 +20,22 @@ support specific guarantees.
 
 ### At-Least-Once
 
-The `at-least-once` delivery guarantee ensures that an [event][docs.data-model]
-received by a Vector component will be delivered at least once. While rare, it
-is possible for an event to be delivered more than once. See the
-[Does Vector support exactly once delivery](#does-vector-support-exactly-once-delivery)
-FAQ below).
+The `at_least_once` delivery guarantee ensures that an
+[event][docs.data-model] received by a Vector component will be
+delivered at least once. For a source, this indicates that it will wait
```

Comment:
This may be pedantic, but presumably we can't guarantee that it will be delivered, but we can guarantee that if it isn't delivered the source will return an error? Eg, if I try to push to the Datadog sink but I don't have an account, it will never be delivered, despite our guarantee. What happens in that situation?

Reply:
There are two senses to the term "delivered", both an attempt to send the event and the confirmation of receipt. This is trying to explain that the sink will attempt delivery until a confirmation or error is received, and that sources will wait for that confirmation before acknowledging. I agree the wording is less than ideal, so I welcome better suggestions.